### PR TITLE
ci: update optimize issue templates to follow monorepo guidelines

### PR DESCRIPTION
**Pull request by misiekhardcore**
## Description

We need to update optimize issue templates to follow monorepo guidelines/standards

## Related issues

relates to https://github.com/camunda/camunda-optimize/issues/13885

### CODE REVIEW

**Review by buccarel — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/optimize-epic.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
 
-name: Optimize Epic
-about: This issue is used to track the progress of an Optimize Epic
-title: ''
-labels: type:epic
-assignees: ''
+name: Optimize PDP Epic
+about: Summary of tasks/features to implement PDP epic.
```
Comment (RomanJRW) [inline .github/ISSUE_TEMPLATE/optimize-epic.md:4]: I am not sure that this is an improvement, it doesn't better describe its purpose

**Review by RomanJRW — CHANGES_REQUESTED** [review]

**Comment by RomanJRW** [comment]
As a side note, please ensure that merge strategy (i.e. target branch to this one, or post-merge to `main`) is coordinated with Mattia

File: .github/ISSUE_TEMPLATE/optimize-epic.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
 ---
 
-name: Optimize Epic
-about: This issue is used to track the progress of an Optimize Epic
-title: ''
-labels: type:epic
-assignees: ''
+name: Optimize PDP Epic
```
[the remark this answers is not in the document]
Comment (misiekhardcore) [reply .github/ISSUE_TEMPLATE/optimize-epic.md:3]: I maybe too blindly wanted to follow templates from operate/tasklist. Let's then keep our old titles and descriptions

File: .github/ISSUE_TEMPLATE/optimize-epic.md (hunk 1 above)
Comment (misiekhardcore) [reply .github/ISSUE_TEMPLATE/optimize-epic.md:4]: the reason is as mentioned in the answer to the other comment

**Comment by misiekhardcore** [comment]
@buccarel how do you want this merged? into the `merge-optimize-in` before you merge it to `main` or you want this PR target `main` and be merged after merging Optimize?

File: .github/ISSUE_TEMPLATE/optimize-epic.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
 ---
-
 name: Optimize Epic
 about: This issue is used to track the progress of an Optimize Epic
-title: ''
-labels: type:epic
-assignees: ''
--------------
+title: 🎯
```
Comment (RomanJRW) [inline .github/ISSUE_TEMPLATE/optimize-epic.md:4]: what is this title for?

**Review by RomanJRW — APPROVED** [review]
Pre-approve, one tiny question

File: .github/ISSUE_TEMPLATE/optimize-epic.md (hunk 3 above)
Comment (misiekhardcore) [reply .github/ISSUE_TEMPLATE/optimize-epic.md:4]: I took it from operate/tasklist tempaltes to make it similar

**Pull request merged**